### PR TITLE
Feature/1.4.4 releasenotes

### PR DESCRIPTION
**Pull request by kuenishi**
Japanese translation will follow once notes fixed.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
-** Bugs Fixed
+# Riak CS 1.4.4 Release Notes
+
+This is a bugfix release. The major fixes are to the storage calculation.
+
+## Bugs Fixed
+
+* create basho-patches directory [riak_cs/775](https://github.com/basho/riak_cs/issues/775)
+
+* `sub_bucket` timeout crashes all storage calculation fixed by [riak_cs/759](https://github.com/basho/riak_cs/issues/759)
```
Comment (shino) [inline RELEASE-NOTES.md]: s/sub/sum/

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
+
+* Access archiver crash fixed by [riak_cs/758](https://github.com/basho/riak_cs/issues/758)
+
+* Throttling access archiver fixed by [riak_cs/747](https://github.com/basho/riak_cs/issues/747)
+
```
Comment (shino) [inline RELEASE-NOTES.md:14]: These two links are opposite?

**Comment by shino** [comment]
All items are covered, nice in general!  Only some tiny comments added.

**Comment by shino** [comment]
:+1: from me :)

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
+This is a bugfix release. The major fixes are to the storage calculation.
+
+## Bugs Fixed
+
+* Create basho-patches directory [riak_cs/775](https://github.com/basho/riak_cs/issues/775) .
+
+* `sum_bucket` timeout crashes all storage calculation is fixed by [riak_cs/759](https://github.com/basho/riak_cs/issues/759) .
+
+* Throttling access archiver is fixed by [riak_cs/758](https://github.com/basho/riak_cs/issues/758) .
```
Comment (andrewjstone) [inline RELEASE-NOTES.md]: - Failure to throttle access archiver is fixed by [riak_cs/758](https://github.com/basho/riak_cs/issues/758) .

**Comment by andrewjstone** [comment]
1 minor clarification. Other than that :+1:

**Comment by kuenishi** [comment]
Thanks, @andrewjstone !

**Comment by kuenishi** [comment]
I forgot to mention that with this change I changed the format from org mode to markdown, and Riak's release notes style, where GH issues or detailed description appended to each item.

**Comment by kuenishi** [comment]
If that change causes any problem please ping me, I can change back the format.

**Comment by andrewjstone** [comment]
I like the change @kuenishi . Thanks :)

**Pull request merged**